test/golden_tsickle_test.ts

```diff
@@ -28,8 +28,8 @@ const TEST_FILTER = (() => {
 })();
 
 // If true, update all the golden .js files to be whatever tsickle
-// produces from the .ts source. Do not change this code but run as:
-//     UPDATE_GOLDENS=y gulp test
+// produces from the .ts source. Do not change this code but export
+// UPDATE_GOLDENS=1 before bazel running the target
```

Comment:
can you run as `UPDATE_GOLDEN=1 bazel run ...`? Please document the full command, it's nice if you can just copy&paste it.

Reply:
Done.